manifests/init.pp

```diff
@@ -0,0 +1,176 @@
+  Optional[Array[String[1]]]  $ignore_prelink_dep_err            = undef,
+  Integer[0,1]                $immutable_set                     = 0,
+  Optional[Array[String[1]]]  $immutwhitelist                    = undef,
+  Optional[Array[String[1]]]  $inetd_allowed_svc                 = undef,
+  Optional[String[1]]         $inetd_conf_path                   = undef,
+  String[1]                   $installdir                        = '/usr',
+  Integer[1]                  $ipc_seg_size                      = 1048576,
+  String[1]                   $language                          = 'en',
+  Integer[1]                  $lock_timeout                      = 300,
+  String[1]                   $lockdir                           = '/var/run/lock',
+  String[1]                   $logfile                           = '/var/log/rkhunter/rkhunter.log',
+  Optional[String[1]]         $mail_on_warning                   = undef,
+  String[1]                   $mail_cmd                          = 'mail -s "[rkhunter] Warnings found for ${HOST_NAME}"', # lint:ignore:single_quote_string_with_variables
+  Integer[0,2]                $mirrors_mode                      = 0,
+  Optional[Array[String[1]]]  $missing_logfiles                  = undef,
+  Optional[String[1]]         $modules_dir                       = undef,
+  Optional[String[1]]         $os_version_file                   = undef,
+  String[1]                   $password_file                     = '/etc/shadow',
+  Integer[0,1]                $phalanx2_dirtest                  = 0,
+  Optional[Array[String[1]]]  $pkgmgr_no_vrfy                    = undef,
+  String[1]                   $pkgmgr                            = 'RPM',
+  Optional[Array[String[1]]]  $port_path_whitelist               = undef,
+  Optional[Array[String[1]]]  $port_whitelist                    = undef,
+  Optional[Array[String[1]]]  $pwdless_accounts                  = undef,
+  Optional[String[1]]         $readlink_cmd                      = undef,
+  Integer[0,1]                $rotate_mirrors                    = 1,
+  Optional[Array[String[1]]]  $rtkt_dir_whitelist                = undef,
+  Optional[Array[String[1]]]  $rtkt_file_whitelist               = undef,
+  Enum['THOROUGH','LAZY']     $scan_mode_dev                     = 'THOROUGH',
+  Optional[Enum['THOROUGH']]  $scanrootkitmode                   = undef,
+  String[1]                   $scriptdir                         = '/usr/share/rkhunter/scripts',
+  Optional[Array[String[1]]]  $scriptwhitelist                   = undef,
+  Optional[Array[String[1]]]  $shared_lib_whitelist              = undef,
+  Integer[0,1]                $show_lock_msgs                    = 1,
+  Integer[0,3]                $show_summary_time                 = 3,
+  Integer[0,1]                $show_summary_warning_number       = 0,
+  Integer[0,1]                $skip_inode_check                  = 0,
+  String[1]                   $ssh_config_dir                    = '/etc/ssh',
```

Comment:
You should have a depency on puppetetlabs/stdlib in the metadata.json and use the Stdlib:Absolutepath type for this so that it verifies the entry is a fully qualified path.